nfc.py

```diff
@@ -60,6 +10,13 @@ def find(cls):
 	logging.basicConfig(level=logging.DEBUG)
 	logger = logging.getLogger(__name__)
 
+	try:
+		from usbhandler import UsbDeviceFinder
+	except ImportError as e:
+		if e.name == 'usb':
+			print("'usb' module note found!.\nThe 'pyusb' library is required.")
```

Comment:
1. Use logger (level - error) or print to stderr (`print('...', file=sys.stderr)`).
2. If module other than usb cannot be imported, you'd not print anything - please add `else` clause for such case, i.e.:
```
else:
    raise
```

Reply:
Darn, you are right! Will do that now.